components/puppeteer/actions/get-html/get-html.mjs

```diff
@@ -1,28 +1,27 @@
 import puppeteer from "../../puppeteer.app.mjs";
+import common from "../common/common.mjs";
 
 export default {
```

Comment:
All components could import the common method and call it as their own instead of having to use bind, e.g.
```suggestion
export default {
  ...common,
```
or
```suggestion
export default {
  methods: common.methods,
```
and then, when running, calling the method as
`const url = this.getFilteredUrl();`